graph_based_slam:
    ros__parameters:
      registration_method: "GICP"
      ndt_resolution: 1.5
      voxel_leaf_size: 0.2
      loop_detection_period: 3000
      threshold_loop_closure_score: 1.5
      distance_loop_closure: 30.0
      debug_flag: false
